t/lib/OpenQA/Test/Utils.pm

```diff
@@ -388,6 +388,12 @@ sub setup_fullstack_temp_dir {
     $config =~ s/^#git_auto_clone = .*/git_auto_clone = no/m;
     $config =~ s/^#git_auto_update = .*/git_auto_update = no/m;
     $configdir->child('openqa.ini')->spew($config);
+
+    my $workers_ini_path = $configdir->child('workers.ini')->to_string;
+    my $workers_ini = Config::IniFiles->new(-file => $workers_ini_path);
+    $workers_ini->setval('global', 'CRITICAL_LOAD_AVG_THRESHOLD', 0);
+    $workers_ini->RewriteConfig;
```

Comment:
Why is this required? And doesn't this just change the file to be like it is anyway (as you modified `t/data/workers.ini` directly)?

Reply:
I thought that other tests use this subroutine and they might be overwright the value of it so I'm forcing the value. Do you think is not necessary? is this a better way to handle the main problem or change the workers.ini is enough?

Reply:
But you don't need both changes. Either change the original file, or change it in `setup_fullstack_temp_dir`.
And if a test is explicitly overwriting that setting, it probably does it for a reason.

Reply:
Yes, make sense. Updating